odl/operator/operator.py

```diff
@@ -240,7 +240,7 @@ def __init__(self, domain, range, linear=False):
         self._is_linear = bool(linear)
         self._is_functional = isinstance(range, Field)
 
-        if self.is_linear:
+        if self._is_linear:
```

Comment:
Is this an optimization? I think we need a policy on this. I in general try to call the property since this makes the code more robust, but there is a performance penalty. You seem to like to call the private variable.